codebase/scss/components/_logo.scss

```diff
@@ -1,3 +1,4 @@
+// Not sure where this is used, afraid to take it out
 #lower-logo h1 a {
-    border-bottom: none;
+    border-bottom: 0;
```

Comment:
This was because the bottom border of the logo was picking up a limegreen border since it has a link. Hack to keep it from having it.